Runtime/BlockadeLabsSkyboxGenerator.cs

```diff
@@ -667,105 +663,121 @@ internal async Task DownloadResultAsync(GetImagineResult result, bool setSkybox
                 AssetDatabase.CreateAsset(skyboxAI, $"{folderPath}/{prefix}.asset");
             }
 
+            if (_skyboxMesh != null)
+            {
+                _skyboxMesh.SkyboxAsset = skyboxAI;
```

Comment:
might want to do this after everything is created and assigned, especially if it will have side effects